bot.py

```diff
@@ -384,14 +389,38 @@ async def _init_request(thread: discord.Thread):
     )
     embed.add_field(
         name="Number of members who voted:",
-        value=f"{request.num_users} members"
+        value=f"{request.num_users} member(s)"
     )
-
     vote_message = await thread.send(embed=embed, view=view)
     await vote_message.pin()
 
     app.update_bot_message_id(thread_id, vote_message.id)
 
+async def update_displayed_member_count(): # Called whenever the displayed member count needs to update
+    for request in app.requests.values():
```

Comment:
You should be able to move this function into the VoteView class as a method and fetch only the relevant `thread_id` from `self` instead of looping through every open request.